src/syscall/syscall_nonhosted.go

```diff
@@ -205,3 +205,7 @@ type Timeval struct {
 	Sec  int64
 	Usec int64
 }
+
+func Getpagesize() int {
+	return 4096 // TODO
```

Comment:
Hmm. I'd say this is not TODO, it's simply unavailable on most baremetal systems because there is no virtual memory and thus no pages. Returning something like 4 or 8 might be more appropriate but I think 4096 is fine for compatibility.
I think putting something like that in a comment would be better than simply TODO (because there is nothing to do here).

Reply:
Done.